common/components/componentsBySection/FindProjects/Filters/RenderFilterCategory.jsx

```diff
@@ -0,0 +1,127 @@
+// @flow
+
+import React from 'react';
+import _ from 'lodash';
+import GlyphStyles from '../../../utils/glyphs.js'
+
+const categoryDisplayNames = {
+  //TODO: move to global constants file
+  'Issue(s) Addressed': "Issue Areas",
+  'Role': "Roles Needed",
+}
+
+//define CSS classes, keep it readable
+const classCategoryExpanded = 'ProjectFilterContainer-category ProjectFilterContainer-expanded';
+const classCategoryCollapsed = 'ProjectFilterContainer-category ProjectFilterContainer-collapsed';
+const classSubcategoryExpanded = 'ProjectFilterContainer-subcategory ProjectFilterContainer-expanded';
+const classSubcategoryCollapsed = 'ProjectFilterContainer-subcategory ProjectFilterContainer-collapsed';
+
+
+class RenderFilterCategory<T> extends React.PureComponent<Props<T>, State> {
```

Comment:
Should at least define what the state contains for this component.  Personally, I would put the expanded/unexpanded states into their own object within state in the event I later needed to put other values in the state.